.scripts/ci.yml

```diff
@@ -279,28 +279,81 @@ stages:
-          - script: npm run test
+          - script: npm run test:rlc
             workingDirectory: $(Build.SourcesDirectory)/packages/typespec-ts
             displayName: "Generate RLC Test Clients from Cadl-ranch"
           - script: npm run check:tree
             workingDirectory: $(Build.SourcesDirectory)/packages/typespec-ts
             displayName: "Check git Tree"
-          - script: npx cadl-ranch check-coverage ./node_modules/@azure-tools/cadl-ranch-specs/http --coverageFiles $(TypeSpec_TESTSERVER_COVERAGE_RLC_AZURE_REPORT) --coverageFiles $(TypeSpec_TESTSERVER_COVERAGE_MODULAR_AZURE_REPORT) --coverageFiles $(TypeSpec_TESTSERVER_COVERAGE_RLC_STANDARD_REPORT) --coverageFiles $(TypeSpec_TESTSERVER_COVERAGE_MODULAR_STANDARD_REPORT)
-            displayName: "Check Coverage"
+          - script: npx tsp-spector check-coverage ./node_modules/@typespec/http-specs/specs --coverageFiles ./coverage/cadl-ranch-coverage-typescript-rlc-standard.json
+            displayName: "Check Coverage RLC Standard"
+            workingDirectory: $(Build.SourcesDirectory)/packages/typespec-ts
+            condition: eq(variables['Agent.OS'], 'Linux')
+            continueOnError: true
+          - script: npx tsp-spector check-coverage ./node_modules/@azure-tools/azure-http-specs/specs --coverageFiles ./coverage/cadl-ranch-coverage-typescript-rlc-azure.json
+            displayName: "Check Coverage RLC Azure"
             workingDirectory: $(Build.SourcesDirectory)/packages/typespec-ts
+            condition: eq(variables['Agent.OS'], 'Linux')
             continueOnError: true
           - publish: $(TypeSpec_TESTSERVER_COVERAGE_RLC_AZURE_REPORT)
             artifact: RLCAzureCoverageReport
             displayName: "Publish rlc azure coverage report"
             condition: and(ne(variables['Build.Reason'], 'PullRequest'), eq(variables['System.TeamProject'], 'internal'), eq( variables['Agent.OS'], 'Linux'))
-          - publish: $(TypeSpec_TESTSERVER_COVERAGE_MODULAR_AZURE_REPORT)
-            artifact: ModularAzureCoverageReport
-            displayName: "Publish modular azure coverage report"
-            condition: and(ne(variables['Build.Reason'], 'PullRequest'), eq(variables['System.TeamProject'], 'internal'), eq( variables['Agent.OS'], 'Linux'))
           - publish: $(TypeSpec_TESTSERVER_COVERAGE_RLC_STANDARD_REPORT)
             artifact: RLCStandardCoverageReport
             displayName: "Publish rlc standard coverage report"
             condition: and(ne(variables['Build.Reason'], 'PullRequest'), eq(variables['System.TeamProject'], 'internal'), eq( variables['Agent.OS'], 'Windows'))
+          - ${{if and(ne(variables['Build.Reason'], 'PullRequest'), eq(variables['System.TeamProject'], 'internal'))}}:
+              - task: AzureCLI@2
+                displayName: "Upload to Cadl Ranch Coverage RLC Report"
+                inputs:
+                  azureSubscription: "Cadl Ranch Storage"
```

Comment:
```suggestion
                  azureSubscription: "TypeSpec Storage"
```

Reply:
should also update here https://github.com/Azure/autorest.typescript/pull/2936/files#diff-622bb099f538b461e78d4aa7c49a82624df09ac8ce2db6bec1227a3ee3df7865R365